patch_api/api/estimates_api.py

```diff
@@ -40,6 +40,8 @@ class EstimatesApi(object):
         "model",
         "make",
         "year",
+        "type",
+        "country",
```

Comment:
🤔  should this here too?
is the generator adding this filter parameter to all the endpoints or just projects?

Reply:
Updated!